tools/reviewer/ReviewerConfigTest.java

```diff
@@ -53,5 +55,12 @@ public void setup() {
   public void protoFileTest() throws Exception {
     fileUtils.readPrototxt("reviewer_config.prototxt", ReviewerConfig.newBuilder());
   }
+
+  @Test
+  public void TotalCrystalCorrectTest() {
+    User user =
```

Comment:
What you want here, is to read the proto:
`ReviewerConfig config = (ReviewerConfig)fileUtils.readPrototxt("reviewer_config.prototxt", ReviewerConfig.newBuilder());`
Then, check that `total_crystal` and the sum of all `crystals` for all users, are equal.